py/farm_ng/core/events_file_reader.py

```diff
@@ -73,6 +81,10 @@ def file_length(self) -> int:
     def file_name(self) -> Path:
         return self._file_name
 
+    @property
+    def event_index(self) -> List[EventLogPosition]:
+        return self._event_index
```

Comment:
Clearer that it is all events, not the index of a single event
```suggestion
    def events_index(self) -> List[EventLogPosition]:
        return self._events_index
```